components/link.jsx

```diff
@@ -0,0 +1,13 @@
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+
+var Linker = React.createClass({
```

Comment:
Yeah, I hate remembering how to get to Link every time I need it, which is often, usually requiring me to open up another .jsx file that uses Link and copying and pasting.

Also, I mean to modify this to swap between <a href=""> and <Link to=""> depending on if the link is internal or external. Something like this: https://github.com/mozilla/teach.webmaker.org/blob/develop/components/icon-link.jsx#L6-L22